Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2505,6 +2505,21 @@ def migrate_last_run(last_run: dict[str, str] | list[dict]) -> list[dict]:
         return [updated_last_run_detections, updated_last_run_incidents, {}]
 
 
+def sort_incidents_summaries_by_ids_order(ids_order, full_incidents, id_field):
```

Comment:
maybe change to this because it should work on both?
```suggestion
def sort_summaries_by_ids_order(ids_order, full_incidents, id_field):
```